server/models/mention-model.js

```diff
@@ -0,0 +1,50 @@
+const mongoose = require("mongoose");
+mongoose.set('useFindAndModify',false);
+const mentionSchema = new mongoose.Schema({
+  content: {
+    type: String,
+    
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  platform: {
+    type: String,
+    required: true,
+  },
+ 
+  image: {
+    type: String,
+  },
+  date: {
+    type: String,
+    required: true,
+  },
+  popularity: {
+    type: String,
+  },
+  url: {
+    type:String,
+  }
+
+});
+const mention = mongoose.model('mention', mentionSchema)
+
+
+async function createMention(data){
+  const newMention = new mention({content: data.content, title: data.title, platform: data.platform, image: data.image,date: data.date, popularity:data.popularity, url: data.url});
```

Comment:
That's awesome!
If you wanted to save yourself some typing, you could have destructured the `data` object above, and done something like this:

```javascript
const { content, title, platform, image, date, popularity, url } = data;
const newMention = new mention({ content, title, platform, image, date, popularity, url });
```

Also, it's usually a good idea to name the constructors with the first letter in uppercase. It's a way to let other devs know that they're dealing with a class/constructor, and that they have to instantiate it using the `new` keyword. So `Mention` instead of just `mention`. The way you're doing it works well, and not capitalizing it won't break anything, but it's just not a [best practice](https://stackoverflow.com/questions/1564398/javascript-method-naming-lowercase-vs-uppercase/1564489#:~:text=5%20Answers&text=A%20popular%20convention%20in%20Javascript,mistakenly%20called%20%22classes%22\).&text=Anything%20that's%20not%20a%20constructor%20usually%20starts%20with%20lowercase%20and%20is%20camelCased.).